scripts/zones/Misareaux_Coast/Zone.lua

```diff
@@ -51,22 +52,20 @@ zoneObject.onEventFinish = function(player, csid, option)
 end
 
 zoneObject.onZoneWeatherChange = function(weather)
-    if os.time() > GetMobByID(ID.mob.ODQAN):getLocalVar("odqanRespawn") and weather == xi.weather.FOG then
-        local chooseOdqan = GetMobByID(ID.mob.ODQAN):getLocalVar("chooseOdqan")
-        local count = 1
+    local odqan = ID.mob.ODQAN
 
-        for k, v in pairs(ID.mob.ODQAN_PH) do
-            if count == chooseOdqan then
-                DisallowRespawn(k, true)
-                DisallowRespawn(v, false)
-                local pos = GetMobByID(k):getSpawnPos()
-                DespawnMob(k) -- Ensure PH is not up
-                GetMobByID(v):setSpawn(pos.x, pos.y, pos.z)
-                SpawnMob(v) -- Spawn Odqan
-            else
-                count = count + 1
-            end
-        end
+    if os.time() > GetServerVariable(string.format("[SPAWN]%s", odqan)) and weather == xi.weather.FOG then
```

Comment:
I see in Frank's recent PR he had to fix this Spawn variable? He had to do something like `\\[Spawn\\]`.